llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -31345,7 +31392,8 @@ static SDValue LowerBITREVERSE(SDValue Op, const X86Subtarget &Subtarget,
   // If we have GFNI, we can use GF2P8AFFINEQB to reverse the bits.
   if (Subtarget.hasGFNI()) {
```

Comment:
Do we need to check AVX/AVX512 feature here?

Reply:
From what I understand, GFNI can be supported even without AVX. So we don't need to have a check for AVX here. 
https://community.intel.com/t5/Intel-ISA-Extensions/Requirements-for-GFNI-without-AVX-AVX-512/m-p/1228325?profile.language=en&countrylabel=Argentina

Reply:
Don't forget Tremont :)